spec/inject_method_spec.rb

```diff
@@ -0,0 +1,12 @@
+require './lib/inject_method' 
+
+describe My_Array do
+
+	it 'is expected to give the same result for #challenge method and inject method' do
```

Comment:
Line is too long. [84/80]<br>Use 2 (not 1) spaces for indentation.<br>Tab detected.